modules/custom/az_migration/src/Plugin/migrate/process/EntityEmbedProcess.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use Drupal\Core\Database\Database;
+
+/**
+ * Process Plugin to recognize text formats and return a given response.
+ *
```

Comment:
I think we should probably add some inline documentation before merging. (At least a list of config options, if any, and a code example)

Reply:
Added @joeparsons suggestion of documentation of usage as well as addition of configuration options that allow for specifying and overriding information for custom content migrations.